index.html

```diff
@@ -6348,6 +6348,41 @@ <h2>Acknowledgements</h2>
 standards process.
     </p>
 
+    <p>
+The Editors gratefully acknowledge the work that led to the creation of this
+specification, and extend a sincere appreciation to those individuals that
+worked on technologies and specifications that deeply influenced this Working
+Group. Namely, the work of Phil Zimmerman, Jon Callas, Lutz Donnerhacke, Hal
+Finney, David Shaw, and Rodney Thayer on <a
+href="https://en.wikipedia.org/wiki/Pretty_Good_Privacy"> Pretty Good Privacy
+(PGP)</a> in the 1990s and 2000s.
+    </p>
+
+    <p>
+In the mid-2010s, preliminary implementations of what would become Decentralized
+Identifiers were <a
+href="https://web-payments.org/minutes/2014-05-07/#topic-1">built</a> in
+collaboration with Jeremie Miller's Telehash project and the W3C Web Payments
+Community Group's work led by Dave Longley and Manu Sporny.
+Some of the <a
+href="https://github.com/WebOfTrustInfo/rwot2-id2020/blob/master/final-documents/requirements-for-dids.pdf">first
+written papers</a> exploring the concept of Decentralized Identifiers can be
+traced back to the first several Rebooting the Web of Trust workshops convened
+by Christopher Allen. That work led to a key collaboration between Christopher
+Allen, Drummond Reed, Les Chasen, Manu Sporny, and Anil John. Anil saw promise
+in the technology and allocated the initial set of government funding to explore
+the space. Without the support of Anil John and his guidance through the years,
+it is unlikely that Decentralized Identifiers would be where they are today.
+Further refinement at the Rebooting the Web of Trust workshops led to Drummond
+Reed producing the <a
+href="https://github.com/WebOfTrustInfo/rwot3-sf/blob/master/topics-and-advance-readings/did-spec-working-draft-03.md">first
+implementers documentation</a> with contributions from Les Chasen, Christopher
+Allen, Ryan Grant, Manu Sporny, Dave Longley, Jason Law, Daniel Hardman, and
+Markus Sabadello. This initial work was then merged into the W3C Credentials
```

Comment:
As with the other links to RWOT documents, let's use the final version rather than the rough draft.  As with the attribution theme that led the WG to consider this section, let's use the available evidence, which here is the author and contributor list at the end of the document.

```suggestion
Further refinement at the Rebooting the Web of Trust workshops led to the <a
href="https://github.com/WebOfTrustInfo/rwot3-sf/blob/master/final-documents/did-implementer-draft-10.pdf">first
implementers documentation</a>, edited by Drummond Reed, Les Chasen, Christopher Allen, 
and Ryan Grant. Contributors included Manu Sporny, Dave Longley, Jason Law, Daniel Hardman,
Markus Sabadello, Christian Lundkvist, and Jonathan Endersby. 
This initial work was then merged into the W3C Credentials
```

Reply:
LGTM.